frontend/frontend/src/store.js

```diff
@@ -20,6 +20,7 @@ export const store = new Vuex.Store({
       username: "",
       email: "",
```

Comment:
Since they don't exist, `null `would be more appropriate initial value than empty string